android/src/main/java/com/bitmovin/player/reactnative/services/MediaSessionPlaybackService.kt

```diff
@@ -1,15 +1,15 @@
 package com.bitmovin.player.reactnative.services
 
 import android.content.Intent
-import android.os.Binder
 import android.os.IBinder
 import com.bitmovin.player.api.Player
 import com.bitmovin.player.api.media.session.MediaSession
 import com.bitmovin.player.api.media.session.MediaSessionService
+import com.bitmovin.player.reactnative.PlayerServiceBinder
 
 class MediaSessionPlaybackService : MediaSessionService() {
```

Comment:
### Open Question about Services Design

Unfortunately, I don't see a clean way of unifying the two services (maybe someone more proficient in Kotlin can help tho 🙏 )

I see mainly 2 drawbacks when unifying the services:
- a media session is always initialized in `onCreate`. Even if I pass data via the intent, `onBind` will be called after `onCreate` from my understanding, so the 'fake' media session is already created
- here we inherit from `MediaSessionService()`. This is not necessary for simple background-playback. Simple backgorund playback, on the other hand, only needs `Service()`, without the additional wrappers